src/js/components/picklist/index.js

```diff
@@ -53,14 +54,27 @@ const PicklistEmpty = View.extend({
   className: 'picklist__message',
   template: hbs`
     {{#if isLoading}}
-      {{ loadingText }}
+      <div class="picklist__message-loading js-loading">
+        {{fas "circle"}}{{ loadingText }}
+      </div>
```

Comment:
This makes this the loading ui/styling for any picklist with the `isListsAsync: true` option. I figured this is fine for now. And we can figure out a way to customize via an options item it in the future if the need arises. Like if we want to use a different icon, spacing, or something across different picklists.

Reply:
Other way would be to pass a `loadingTemplate`option or something similar that's handlebars template.